www/quiz_style.css

```diff
@@ -151,21 +151,21 @@
   overflow-y: hidden;
 }
 .mdbook-quiz .prompt .bug-report {
-  position: absolute;
-  max-width: 70%;
-  right: 0.5em;
-  top: 0.5em;
-  text-align: right;
+  display: none;
+  /* position: absolute; */
```

Comment:
If these are going to get commented out, just remove them in this PR

Reply:
Done; I took out all the `.bug-report` styles